src/modules/edc-demo/components/contract-agreement-transfer-dialog/contract-agreement-transfer-dialog.component.html

```diff
@@ -52,7 +239,7 @@ <h1 mat-dialog-title>Initiate Transfer</h1>
   <button
     mat-raised-button
     color="primary"
-    [disabled]="!form.formGroup.valid || loading"
+    [disabled]="!form.datasink.valid || loading"
```

Comment:
Since you renamed formGroup to all, this especially needs to be "!form.all.valid", since you want to block submits if the entire form is not valid